app/controllers/api/v1/activity_sessions_controller.rb

```diff
@@ -75,7 +75,7 @@ def find_activity_session
   def activity_session_params
     params.delete(:activity_session)
     @data = params.delete(:data)
-    concept_tag_keys = [:concept_tag_name, :metadata => concept_tag_result_allowed_keys]
+    concept_tag_keys = [:concept, :metadata => concept_result_allowed_keys]
```

Comment:
also, can well call the key for the concept uid 'concept_uid' instead of 'concept'?